docs/resources/gcp_cloud_configuration.md

```diff
@@ -0,0 +1,45 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "env0_gcp_cloud_configuration Resource - terraform-provider-env0"
+subcategory: ""
+description: |-
+  configure a GCP cloud account (Cloud Compass)
+---
+
+# env0_gcp_cloud_configuration (Resource)
+
+configure a GCP cloud account (Cloud Compass)
```

Comment:
Nice job here, I see we are lacking these docs for Azure so good to have it like AWS.
<img width="355" alt="image" src="https://github.com/user-attachments/assets/a820f669-adf8-49cc-b2f9-5712a6677964" />

Reply:
https://env0.slack.com/archives/C01QBBEGBEH/p1750586653695669